src/packages/@ncigdc/utils/trace.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+export default C => p => (
+  console.info(
+    `%ctracing ${C.displayName} props:\n==========================`,
+    'color: rgb(22, 138, 96);font-weight: bold;',
+  ), // eslint-disable-line
+  console.log(p), // eslint-disable-line
+  <C {...p} />
```

Comment:
`C(p)` will fail if it's a class. that's the one use case where JSX aka `React.createElement` is better. I'll make the names better though